ui/ducks/send/send.js

```diff
@@ -1401,6 +1406,16 @@ const slice = createSlice({
 
       state.gas.error = insufficientFunds ? INSUFFICIENT_FUNDS_ERROR : null;
     },
+    validateUserInputHexData: (state) => {
+      const isValidHex =
+        isHexString(state.draftTransaction.userInputHexData) ||
+        !state.draftTransaction.userInputHexData;
+      if (isValidHex) {
+        state.draftTransaction.error = null;
+      } else {
+        state.draftTransaction.error = INVALID_HEX_STRING_ERROR;
+      }
```

Comment:
there's no longer a draftTransaction object.
suggestion:
```js
hexData: {
  input: '',
  error: null,
}
```
then modify the hexData.input key and put the error on that hexData.error key.